sdks/python/src/opik/api_objects/trace/helpers.py

```diff
@@ -0,0 +1,59 @@
+from . import trace
+from .. import span, helpers
+from ... import id_helpers
+from datetime import datetime
+import uuid
+from typing import Optional, List, Tuple
+import logging
+
```

Comment:
Let's call this module `migration` instead of `helpers`, it's pretty narrow-aimed.